src/doc/trpl/guessing-game.md

```diff
@@ -95,8 +95,9 @@ import the `std::old_io` module with `use`, and then our `main` function contain
 our program's logic. We print a little message announcing the game, ask the
 user to input a guess, get their input, and then print it out.
 
-Because we talked about this in the section on standard I/O, I won't go into
-more details here. If you need a refresher, go re-read that section.
+Because we talked about this in [the section on standard
+I/O](standard-input.html), I won't go into more details here. If you need a
+refresher, go re-read that section.
```

Comment:
Yeah, I"ve historically preferred the inline style, but the documentation style RFC which recently landed mandates the opposite, so new stuff is written in that style.